### ISSUE TITLE
Scarches refactor controlling

### ISSUE THREAD

**Issue by Cottoneyejoe95** [theislab/scarches#62]
Would be cool if another pair of eyes would check the functionality of my refactor branch. If everything works I could push to master and release a new version. 
`marco/trvae_refactor` is the name of the branch.

**Comment by M0hammadL** [comment]
@Koncopd could you please have alook!?

**Comment by Koncopd** [comment]
Yep, i'll look.

**Comment by Koncopd** [comment]
@Cottoneyejoe95 pls create the pr for this.

**Comment by Cottoneyejoe95** [comment]
@Koncopd Created pr :)

**Comment by M0hammadL** [comment]
is there any progress here?

**Comment by Cottoneyejoe95** [comment]
I dont think any of my changes are in master yet. @Koncopd I also made a new change to my branch yesterday which is necessary for the master branch. Should I make a new pr?

**Comment by Koncopd** [comment]
Yep, i will check soon, sorry for the delay.
@Cottoneyejoe95 Are all fixes in the pr now?

**Comment by Cottoneyejoe95** [comment]
No problem. The newest fixes are only in my branch but not in the PR, since vodafone has a technical failure and I have no internet today...atleast not until now

**Comment by Cottoneyejoe95** [comment]
Just saw that I can also do it with my phone. So now there is an updated PR.

**Comment by Cottoneyejoe95** [comment]
@Koncopd Should we do a new release then?

**Comment by Koncopd** [comment]
I also want to merge [adaptors](https://github.com/theislab/scarches/pull/50) before the release.

**Comment by Cottoneyejoe95** [comment]
@Koncopd @M0hammadL whats the state of the adaptors? we should really consider releasing a new version, since a lot of updates happened since February.

**Comment by M0hammadL** [comment]
let's release , adaptors take time.

**Comment by Koncopd** [comment]
I am merging adaptors and will add a tutorial later, this won't require a new release.
